extensions/amp-apester-media/0.1/monetization/companion/display.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {!JsonObject} media
+ * @param {AmpElement} apesterElement
+ */
+export function handleCompanionDisplay(media, apesterElement) {
+  const monetizationSettings = media['campaignData'] || {};
+  const companionRawSettings = monetizationSettings['companionOptions'] || {};
+  const {enabled, settings = {}} = companionRawSettings || {};
```

Comment:
companionRawSettings already has a fallback of `{}`, it seems like there is another (unneeded) fallback here.

Reply:
fixed